src/core_plugins/timelion/public/directives/timelion_expression_input_helpers.js

```diff
@@ -73,6 +73,26 @@ function inLocation(cursorPosition, location) {
   return cursorPosition >= location.min && cursorPosition <= location.max;
 }
 
+function getArgumentsHelp(functionHelp, functionArgs = []) {
+  if (!functionHelp) {
+    return [];
+  }
+
+  // Do not provide 'inputSeries' as argument suggestion for chainable functions
+  const argsHelp = functionHelp.chainable ? functionHelp.args.slice(1) : functionHelp.args.slice(0);
+
+  // ignore arguments that are all ready provided in function declaration
+  const providedArguments = functionArgs.map((arg) => {
+    return arg.name;
+  });
+  return argsHelp.filter(arg => {
+    if (providedArguments.includes(arg.name)) {
```

Comment:
Since `includes` returns a boolean already, I think we can simplify this to:

```js
  return argsHelp.filter(arg => {
    return !providedArguments.includes(arg.name);
  });
```